SETUP.md

```diff
@@ -1,3 +1,20 @@
+- [Cloning the manubot/rootstock repository to create a new manuscript](#cloning-the-manubot-rootstock-repository-to-create-a-new-manuscript)
```

Comment:
The table of contents is a great addition.  I was testing it and noticed GitHub sanitizes `manubot/rootstock` differently here:
`#cloning-the-manubotrootstock-repository-to-create-a-new-manuscript`

We can touch that up in a future pull request.